extensions-marketplace/develop/integrate/faceunity.mdx

```diff
@@ -0,0 +1,482 @@
+---
+title: "FaceUnity"
+sidebar_position: 3
+type: docs
+description: >
+    Integrate and use the FaceUnity AR Filter extension in your  Android or iOS app.
+---
+export const toc = [{}];
+
+This page introduces how to integrate and use the FaceUnity AR Filter extension in your  Android or iOS app.
+
+## Understand the tech
+
+The FaceUnity AR Filter extension encapsulates the core APIs of the FaceUnity Nama SDK. By calling the <Link to="{{Global.API_REF_ANDROID_ROOT}}/class_irtcengine.html#api_setextensionproperty">setExtensionProperty</Link>
```

Comment:
This sentence seems to be redundant. It breaks the flow

Reply:
Done.